packages/occ-worker/src/occapi.ts

```diff
@@ -349,6 +347,40 @@ export function _loadBrep(arg: {
   return shape;
 }
 
+export function _loadStepFile(content: string): OCC.TopoDS_Shape | undefined {
+  const oc = getOcc();
+  const fakeFileName = `${uuid()}.STEP`;
+  oc.FS.createDataFile('/', fakeFileName, content, true, true, true);
+
+  const reader = new oc.STEPControl_Reader_1();
+
+  const readResult = reader.ReadFile(fakeFileName);
+
+  if (readResult === oc.IFSelect_ReturnStatus.IFSelect_RetDone) {
+    reader.TransferRoots(new oc.Message_ProgressRange_1());
+    const shape = reader.OneShape();
+    oc.FS.unlink('/' + fakeFileName);
+    return shape;
+  } else {
+    console.error('Something in OCCT went wrong trying to read');
+    return undefined;
+  }
+}
+
+export function _loadObjectFile(arg: {
+  content: string;
+  type: string;
```

Comment:
a more specific interface in form of `step | brep` would be better. I goes with the comment in the `any.json` schema.